api/openapi.yaml

```diff
@@ -2093,10 +2160,8 @@ components:
           # on the client side.
           #
           # readOnly: true
-        scan:
-          $ref: '#/components/schemas/ScanRelationship'
-        asset:
-          $ref: '#/components/schemas/TargetRelationship'
+        assetScan:
+          $ref: '#/components/schemas/TargetScanResultRelationship'
```

Comment:
why we need this on finding? 
why `finding` is holding the `summary` of the scan and `maxParallelScanners` for example?

Reply:
This is a the assetScan which found the finding, a finding shouldn't have a relationship to a scan directly because going forward AssetScans could exist without a Scan.

Reply:
> a finding shouldn't have a relationship to a scan directly because going forward AssetScans could exist without a Scan

But `TargetScanResultRelationship` holds info that is not relevant to finding, DYT?

Reply:
This is a relationship to the AssetScan/TargetScanResult which found the finding its not embeded information unless you $expand. But personally I think all the AssetScan information is relavant, because you may wish to check the configuration of the scan which found the finding etc.